x-pack/plugins/security_solution/public/flyout/document_details/right/tabs/json_tab.tsx

```diff
@@ -48,31 +48,33 @@ export const JsonTab: FC = memo(() => {
   return (
     <EuiFlexGroup
       ref={flexGroupElement}
-      direction={'column'}
-      gutterSize={'none'}
+      direction="column"
+      gutterSize="none"
       data-test-subj={JSON_TAB_CONTENT_TEST_ID}
     >
       <EuiFlexItem>
         <EuiFlexGroup justifyContent={'flexEnd'}>
           <EuiFlexItem grow={false}>
             <CopyToClipboard
               rawValue={jsonValue}
-              text={
+              data-test-subj={JSON_TAB_COPY_TO_CLIPBOARD_BUTTON_TEST_ID}
```

Comment:
Same as above, I think since we are not dictating the children element, the updated `CopyToClipboard` is very similar to the generic `EuiCopy` + modifier. If the copy function is extracted, this could look like:

```
<EuiCopy textToCopy={jsonValue}>
  {(copy) => (
    <EuiButtonEmpty
      iconType={'copyClipboard'}
      size={'xs'}
      aria-label={i18n.translate(
        'xpack.securitySolution.flyout.right.jsonTab.copyToClipboardButtonAriaLabel',
        {
          defaultMessage: 'Copy to clipboard',
        }
      )}
      data-test-subj={JSON_TAB_COPY_TO_CLIPBOARD_BUTTON_TEST_ID}
      onClick={() => copyFunction(copy, jsonValue)}
      onKeyDown={() => copyFunction(copy, jsonValue)}
    >
      <FormattedMessage
        id="xpack.securitySolution.flyout.right.jsonTab.copyToClipboardButtonLabel"
        defaultMessage="Copy to clipboard"
      />
    </EuiButtonEmpty>
  )}
</EuiCopy>
```

Reply:
done :) Also did the changes to the `header_actions` file